src/pyramid_debugtoolbar/toolbar.py

```diff
@@ -63,8 +63,10 @@ def __init__(
         self.visible = False
 
         # Panels can be be activated (more features) (e.g. Performace panel)
```

Comment:
Would you please fix the spelling of "Performace" to "Performance"? I know it's outside this PR's scope, but I just noticed it during review.